### PR TITLE
Add makefile and tidy code

### PR DESCRIPTION
**Pull request by insacuri**
Added a lint option that does both flake8 and checks for security vulnerabilities.
Updated readme with new ways to do things

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -28,6 +28,7 @@ retrying = "==1.3.3"
 sqlalchemy = "==1.1.10"
 sqlalchemy-utils = "==0.32.14"
 structlog = "==17.2.0"
+"flake8" = "*"
```
Comment (andrewmil) [inline Pipfile]: This should be in `dev-packages` not `packages`

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+build:
+	pipenv install --dev
+
+lint:
+	pipenv run flake8 ./ras_party ./test
+	pipenv check ./ras_party ./test
+
+start:
+	pipenv run python3 run.py
```
Comment (andrewmil) [inline Makefile]: Should we have a command for running tests as well?

**Review by andrewmil — APPROVED** [review]
Looks good 👍 
Just left a couple small comments

File: Pipfile (hunk 1 above)
Comment (insacuri) [reply Pipfile]: Removed, was already in dev-packages anyway

File: Makefile (hunk 2 above)
Comment (insacuri) [reply Makefile]: Added a test command

**Pull request merged**